Moggles/ClientApp/TogglesList.vue

```diff
@@ -435,9 +480,56 @@
 						return true;
 				}
 				return false;
-			},
+            },
+            getAllScheduledToggle(appId) {
+
+                axios.get("/api/toggleScheduler", {
+                    params: {
+                        applicationId: appId
+                    }
+                }).then((response) => {
+
+                    //create the flattened row models
+                    let models = _.map(response.data, toggle => {
+                        let model = {
+                            id: toggle.id,
+                            toggleName: toggle.toggleName,
+                            scheduledDate: toggle.scheduledDate,
+                            scheduledState: toggle.scheduledState,
+                            updatedBy: toggle.updatedBy,
+                            environments: toggle.environments
+                        };
+                        return model;
+                    });
+                    this.scheduledToggles = models;
+                }).catch(() => {
+                    //do not uncomment this, the null reference exception will return to haunt us !
+                    //window.alert(error)
+                });
+            },
+
+            isScheduled(toggleName) {
+                let filtered = _.map(this.scheduledToggles.filter(ft => ft.toggleName == toggleName), sch => {
+                    let model = {
+                        toggleId: sch.id,
+                        toggleName: sch.toggleName,
+                        scheduledDate: sch.scheduledDate,
+                        scheduledState: sch.scheduledState,
+                        updatedBy: sch.updatedBy,
+                        environments: sch.environments
+                    };
+                    return model;
+                });
+                return filtered;
+            },
+            editToggleScheduler(toggle) {
+                Bus.$emit("edit-schedule", toggle.toggleId);
+                this.showScheduler = true;            
+            },
+
 
-			loadGridData(appId) {
+            loadGridData(appId) {
+                Bus.$emit('toggle-scheduled');
```

Comment:
I think you should call getAllScheduledToggle directly without emitting the event even if they do the same thing. Maybe in the future the handler will be modified and it will not apply for this case anymore and because  also the action performed is not resulted after a toggle was scheduled